.env

```diff
@@ -0,0 +1 @@
+REACT_APP_PROXY_TARGET='http://localhost'
```

Comment:
Das hier ist der default, wenn ihr beides lokal laufen habt. Wenn ihr das gegen die Production-API schicken wollt, könnt ihr eine lokale .env Datei anlegen, und das dort überschreiben. Das ist in create-react-app automatisch mit dabei, siehe: https://create-react-app.dev/docs/adding-custom-environment-variables

Das heißt, ihr könnt eine `.env.local` Datei anlegen und dort `REACT_APP_PROXY_TARGET='http://xxx-xxxxx.de'` eintragen, dann wird automatisch dieser Wert genommen. Die URL habe ich bewusst nicht ins Repository genommen, um böse Machenschaften zu vermeiden (z.B. durch Crawler).